package.json

```diff
@@ -265,8 +265,6 @@
 		"onCommand:openshift.component.debug.palette",
 		"onCommand:openshift.resource.load",
 		"onCommand:openshift.resource.unInstall",
-		"onCommand:openshift.service.create",
-		"onCommand:openshift.service.delete",
```

Comment:
I see `openshift.service.delete` is still present in odo.test.ts. Should that be removed as well ?

Reply:
Good catch! Yes, I will remove it in this PR. I'll also remove a similar call to `openshift.service.describe`. I am planning on rewriting odo.test.ts in my PR #2799.